pythontodo/src/todo.py

```diff
@@ -0,0 +1,54 @@
+from sqlalchemy import Column, Integer, String, Boolean, DateTime
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy import create_engine
+from sqlalchemy.orm import sessionmaker
+
+
+Base = declarative_base()
+
+class Todo(Base):
+    __tablename__ = 'todos'
+    
+    id = Column(Integer, primary_key=True, autoincrement=True)
+    task = Column(String)
+    description = Column(String)
+    completed = Column(Boolean)
+    edited = Column(Boolean)
+    serialNumber = Column(Integer)
+    lastViewed = Column(Boolean)
+    createdDate = Column(String)
+    modifiedDate = Column(String)
+    
+
+    
+    def __init__(self, task, description, completed, edited, createdDate, modifiedDate, lastViewed,serialNumber):
+        self.task = task
+        self.description = description
+        self.completed = completed
+        self.edited = edited
+        self.createdDate=createdDate
+        self.modifiedDate=modifiedDate
+        self.lastViewed = lastViewed
+        self.serialNumber = serialNumber
+        
+        engine = create_engine('sqlite:///todos.db')
```

Comment:
we don't need a Database, just do it in memory. Using a database adds complexity that is not needed.

Reply:
This we just added to match with Node server, for next server language, I will try to store it locally without database.